documentation-site/components/yard/config/select.ts

```diff
@@ -0,0 +1,387 @@
+import pick from 'just-pick';
+import {Select, SIZE, TYPE} from 'baseui/select';
+import {PropTypes} from '../const';
+import {changeHandlers} from './common';
+import {TConfig} from '../types';
+
+const selectConfig: TConfig = {
```

Comment:
```suggestion
const SelectConfig: TConfig = {
```

Just to match all other places.